### PR TITLE
fix(content): Tweak "Retired Family to Earth" mission

### PR DESCRIPTION
**Pull request by Pointedstick**
## Summary
After playing the mission, it felt a bit awkward to me.

## Fix Details
The following changes have been made to it:
- Improved the opening to make the mission start less abruptly
- Tweaked Donna's dialogue to make her feel more like a senior citizen
- Reduced the payment to reinforce ithe bitterness of their poverty in
  retirement
- Minor grammar and punctuation improvements

### CODE REVIEW

File: data/human/transport missions.txt [hunk 1]
```diff
@@ -3173,17 +3173,17 @@ mission "Earth Retirement"
...
-			`	"My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us."`
-			`	As Donna talks, Charles begins to frown. "I suppose you could put it like that."`
-			`	"Is that wrong?" Donna replies, who appears to be bracing for the response.`
+			`	You inquire as to what's troubling them, and the woman begins telling you her life's story. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us. Oh, where are my manners? I'm Donna. Pleased to meet you."`
+			`	Donna was clearly looking for someone besides her husband to talk to. But Charles begins to frown. "I suppose you could put it like that."`
```
Comment (W1zrad) [inline data/human/transport missions.txt:3182]:
```suggestion
			`	Donna was clearly looking for someone besides her husband to talk to, but Charles begins to frown. "I suppose you could put it like that."`
```
The pause caused by the period feels a little too long when I read it to myself, but up to you.

Comment (Pointedstick) [reply data/human/transport missions.txt:3182]: Yeah, upon re-reading, I think you're right.

**Review by petervdmeer — APPROVED** [review]
The improvements look good to me.

File: data/human/transport missions.txt [hunk 2]
```diff
@@ -3173,17 +3173,17 @@ mission "Earth Retirement"
...
-			`	"My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us."`
-			`	As Donna talks, Charles begins to frown. "I suppose you could put it like that."`
-			`	"Is that wrong?" Donna replies, who appears to be bracing for the response.`
+			`	You inquire as to what's troubling them, and the woman begins telling you her life's story. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us. Oh, where are my manners? I'm Donna. Pleased to meet you."`
+			`	Donna was clearly looking for someone besides her husband to talk to, but Charles begins to frown. "I suppose you could put it like that."`
```
Comment (Zitchas) [inline data/human/transport missions.txt:3182]:
```suggestion
			`	Donna is clearly looking for someone besides her husband to talk to, but Charles begins to frown. "I suppose you could put it like that."`
```

Seeing as this is describing what is currently going on, shouldn't this be in the present tense?

**Review by Zitchas — CHANGES_REQUESTED** [review]
This does look like a solid improvement. That one was vs is comment aside, looks good.

File: data/human/transport missions.txt (hunk 2 above)
Comment (Pointedstick) [reply data/human/transport missions.txt:3182]: This was intended to be an explanation for why she just told you half her life's story. By the point the player reads this, that's already happened.

Comment (Zitchas) [reply data/human/transport missions.txt:3182]: hmmm. OK.

**Review by Zitchas — APPROVED** [review]
Well, with that one thing out of the way, looks good to me.

File: data/human/transport missions.txt (hunk 2 above)
Comment (Pointedstick) [reply data/human/transport missions.txt:3182]: ok :)

**Comment by Pointedstick** [comment]
Thanks!

**Pull request merged**